docs/installation.md

```diff
@@ -81,6 +81,7 @@ the following third-party contributions:
 
 ### Ansible
 
+* [SoInteractive/ansible-prometheus](https://github.com/SoInteractive/ansible-prometheus)
```

Comment:
I just looked at all of these roles the other day - the SoInteractive one has 6 contributors and is regularly updated. The others not so much. I'd recommend replacing them with this.